edge/pkg/edged/volume/csi/csi_controller.go

```diff
@@ -30,7 +32,7 @@ type Controller struct {
 func (c *Controller) CreateVolume(req *csi.CreateVolumeRequest) (*csi.CreateVolumeResponse, error) {
 	if c.csiClient == nil {
 		var err error
-		c.csiClient, err = newCsiDriverClient(csiDriverName("csi-hostpath"))
+		c.csiClient, err = newCsiDriverClient(csiDriverName(edgedconfig.Config.CsiDriverName))
```

Comment:
Here we only can support one csi driver? @lvchenggang @moolen

Reply:
Is it possible to read the name from some parameters?

Reply:
Yes, right now `edged` can only talk to one specific csi driver. Because the driver name is hard-coded `kubeedge` does 
 support only the `csi-hostpath` driver. But we can potentially support `nfs` or any other driver. 
The csi-driver name can be configured through `/etc/kubeedge/config/edgecore.yaml`.

Reply:
/hold
keadm need to fix also, although this parameter is configurable, keadm should pass in parameter to fix this default value.

Reply:
> We could potentially support multiple csi drivers on the edge side but there is a lot of work to be done in order to support that.

The storage driver name maybe should from volume spec, and then edgecore calls the related driver? Configured from `/etc/kubeedge/config/edgecore.yaml` is not that extensible.

Reply:
> Configured from /etc/kubeedge/config/edgecore.yaml is not that extensible

i agree.

> The storage driver name maybe should from volume spec

Unfortunately it's not embedded in the CreateVolume request. we probably need to embed the `Volume*` with additional metadta (the to-be-called csi driver on the edge)

I opened #3471 to discuss the high-level approach there.

Reply:
Agree with that, the modification of this PR is not extensible.